bundles/org.openhab.ui.basic/src/main/resources/OH-INF/config/config.xml

```diff
@@ -42,6 +42,15 @@
 			</options>
 			<default>false</default>
 		</parameter>
+		<parameter name="webAudio" type="text" required="true">
```

Comment:
You used the same pattern (text type) as for other parameters.
I do not understand why boolean type was not used for enableIcons, condensedLayout, ... parameters.
But your new parameter is at least coherent with others.

Reply:
Yes I kept the new parameter coherent with the other config parameters. It seems that @kaikreuzer added the inital config parameter in https://github.com/eclipse-archived/smarthome/pull/540 like this. Maybe because it easier to understand (compared to a checkbox)?